src/Usb.Net/UsbControlRecipient.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Usb.Net
+{
+    /// <summary>
+    /// Defines constants that indicate the recipient of a USB control transfer. 
+    /// The recipient is defined in the setup packet of the control request. See Table 9.2 of section 9.3 of the Universal Serial Bus (USB) specification (<see cref="www.usb.org"/>).
+    /// </summary>
+    public enum UsbControlRecipient
```

Comment:
@networkfusion question...

Where do these enums and descriptions come from? Is this from some official documentation?